### PR TITLE
updated docs for `at_contact` library

### PR DESCRIPTION
**Pull request by yahu1031**
**- What I did**

- Update the `at_contact` library documentation(README.md) W.R.T flutter

**- How I did it**

- Cloned, created branch, updated.

**- How to verify it**

- Head over to `at_contact` [README.md](https://github.com/atsign-foundation/at_libraries/blob/docs_update/at_contact/README.md)

**- Description for the changelog**

Usage of the library update.

### CODE REVIEW

**Comment by yahu1031** [comment]
I need to get confirmation about the example app for this library.

**Review by murali-shris — APPROVED** [review]

**Comment by murali-shris** [comment]
keep the readme simple with only dart usage..flutter snippets can be part of at_contacts_flutter

**Review by cpswan — APPROVED** [review]

**Review by cpswan — APPROVED** [review]

**Review by CynthiaPeter — COMMENTED** [review]
Can we change "Initially to get a basic overview of the @protocol packages, You must read the [atsign docs](https://atsign.dev/docs/overview/)." to 

- To get a basic overview of the @protocol packages, Please visit the [atsign  developer docs](https://atsign.dev/docs/overview/) to learn more. (Line 16)

- You need some basic setup to use this package. Follow our [get started guide](https://atsign.dev/docs/get-started/setup-your-env/) to set up. (Line 18)

Also, it is important we include tags and keywords.

**Comment by gkc** [comment]
@yahu1031 this was approved but not merged; shall you merge it to trunk?

**Review by gkc — APPROVED** [review]

**Comment by yahu1031** [comment]
@gkc Uh oh! Really sorry I thought it was merged. Lemme just go through the code again cause things have little changes from then to now.

File: at_contact/README.md [hunk 1]
```diff
@@ -19,20 +19,21 @@ Initially to get a basic overview of the @protocol packages, You must read the [
 
 # Usage
 
-- Create an instance of `AtContactsImpl`, We make use of `AtClientManager`.
+- Create an instance of `AtContactsImpl`, We make use of `AtClientManager` for `AtClient` to be passed.
 
 ```dart
 // Create an instace of AtClient.
```
Comment (gkc) [inline at_contact/README.md]: 'Instance' is misspelled as 'instace' in a few places, please correct

**Review by gkc — APPROVED** [review]

**Pull request merged**